### ISSUE TITLE
"i3status adjust_time failed" since update to 2.0

### ISSUE THREAD

**Issue by Cutuchiqueno** [ultrabug/py3status#58]
After I updated py3status to version 2.0 I experience the `adjust_time` error that appeared several times in the issue list for the first time:

```
Jan 09 15:17:14 sputnik3arch /py3status[451]: i3status adjust_time failed (time data '15:03 - 09.01.2015' does not match format '%Y-%m-%d %H:%M:%S')
```

on Arch Linux

```
$ locale
LANG=de_DE.UTF-8
LC_CTYPE=de_DE.UTF-8
LC_NUMERIC="de_DE.UTF-8"
LC_TIME="de_DE.UTF-8"
LC_COLLATE="de_DE.UTF-8"
LC_MONETARY="de_DE.UTF-8"
LC_MESSAGES="de_DE.UTF-8"
LC_PAPER="de_DE.UTF-8"
LC_NAME="de_DE.UTF-8"
LC_ADDRESS="de_DE.UTF-8"
LC_TELEPHONE="de_DE.UTF-8"
LC_MEASUREMENT="de_DE.UTF-8"
LC_IDENTIFICATION="de_DE.UTF-8"
LC_ALL=
```

before I never had it.

Any idea?

**Comment by ultrabug** [comment]
Hi @Cutuchiqueno, damn this problem is annoying and I'm surprised of it because I surely didn't update this part of the code.

So this must have to do with i3status.conf parsing. Would you care to show me the content of your time {} configuration format please ?

Cheers

**Comment by Cutuchiqueno** [comment]
sorry, that was the one I forgot, here it is:
```
tztime local {
        format = "%H:%M - %d.%m.%Y"
}
```

**Comment by ultrabug** [comment]
hmm I still can't reproduce this bug mate.

I configured an arch linux VM with the same locale and the same tztime config and it works.. Can you make sure you're running py3status v2.1 please ?

**Comment by ultrabug** [comment]
I recently merged some work from a fix branch. It looks like you're using py3status-git. Mind re-installing it with latest master please and let me know ?

Else, I'll have to ask you to show me your entire i3status.conf (you can come on IRC to do so or send it by email to me if you prefer it to remain private)

**Comment by Cutuchiqueno** [comment]
In fact I had installed py3status. The line saying py3status-git only says that it (of cause) conflicting with this package, not that it is installed. Sorry I treid to change the langauge output of the package description but did not succeed.

Nevertheless, I removed py3status and replaced it with py3status-git and the problem is gone. So, between 2.1 and the current master branch there is a diff that affects this problem.
